### PR TITLE
DG addtoclassCmd and its UML diagrams

### PR DESCRIPTION
**Pull request by Ongzl**
Also, edited AddClassCMDs UML diagram

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -324,6 +324,41 @@ The AddCommandParse parses the user input to obtain the classTiming (denoted by
 uniquely identifies the class. Afterwards, an AddCommand is created with the `Class` and `Student`, after which
 it checks whether an existing class with the same timing exists and adds the student to the `Class`'s classList
 and if not, adds the student to the new class created.
+### Adding Student(s) to a class feature
+Allows user to add any existing student into any single existing tuition class in order to keep track of which 
+students are being taught in which class.
+
+## Implementation
```
Comment (yourally2) [inline docs/DeveloperGuide.md:331]: Your header sizes are mixed up haha

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -324,6 +324,41 @@ The AddCommandParse parses the user input to obtain the classTiming (denoted by
...
+To add Student(s) to a class, the `addtoclass` command is used. The user input is passed to
+the `LogicManager`, which parses the input using the `TimesTableParser` and `AddToClassCommandParser`.
+A `AddToClassCommand` is then created with the class and student indices involved in the command.
```
Comment (yourally2) [inline docs/DeveloperGuide.md:334]: Small nitpick, I think this should be 'an'

**Review by yourally2 — COMMENTED** [review]
Other than this, LGTM!

**Comment by s7u4rt99** [comment]
LGTM

**Pull request merged**